### ISSUE TITLE
When Compile ukui-search-extension. action-lable.h, there is no such file or directory

### ISSUE THREAD

**Issue by Tyl2020301** [ukui/ukui-search-extensions#4]
When Compile ukui-search-extension. “action-lable.h”, there is no such file or directory。

**Comment by iaom** [comment]
Have you installed ukui-search first?

**Comment by iaom** [comment]
By the way, you have to install the banch "new-frontend".

**Comment by Tyl2020301** [comment]
yeah , ukui-search installed, but still "action-lable.h" not found

**Comment by Tyl2020301** [comment]
@iaom

**Comment by iaom** [comment]
Which branch have you installed?

**Comment by iaom** [comment]
Two version   1. branch new-frontend of ukui-search installed  ------> complie branch main of ukui-search-extensions 
                      2. branch ukss-dev of ukui-search installed  ------> complie branch dev-unity of ukui-search-extensions

**Comment by iaom** [comment]
@Tyl2020301
